Course/Course/Presentation/Container/CourseContainerView.swift

```diff
@@ -316,7 +316,7 @@ public struct CourseContainerView: View {
             }
         }
         .versionedTabStyle()
-        .introspect(.scrollView, on: .iOS(.v15, .v16, .v17), customize: { tabView in
+        .introspect(.scrollView, on: .iOS(.v16, .v17), customize: { tabView in
```

Comment:
@forgotvas  What will happen when the new iOS version iOS 18, will be released, do we need to white list future updates or will it be handled automatically?

Reply:
we will need add new version or we will need to change standard introspect to `@_spi(Advanced) import SwiftUIIntrospect` and use it like `.introspect(.scrollView, on: .iOS(.v15...), customize: { scrollView in`

Reply:
Can you update it so we don't have to worry about future updates, until unless there is an issue?

Reply:
or if it's out of the scope of this PR, we can create a task for this and can handle it post-MVP.

Reply:
updated, also checked via search on project word `.introspect` and found that there no any other places where is uses list like `.v(16), .v(17)`. Other places use certain version or relative range.